CorsixTH/Lua/dialogs/edit_room.lua

```diff
@@ -1141,43 +1145,43 @@ local function validDoorTile(xpos, ypos, player_id, world)
   local th = TheApp.map.th
   local tile_flags = th:getCellFlags(xpos, ypos)
   -- check builable and own it
-  if not (tile_flags.buildable or tile_flags.owner == player_id) then return false end
+  if not tile_flags.buildable and tile_flags.owner ~= player_id then return false end
```

Comment:
Still pretty sure you actually want not buildable or not owned by the player here.

Reply:
I have found a defect with this logic that I will push out.
tile_flags.buildable will be false for every door when a bench footprint 'passable'/shareable tile is in front of the door, it basically only needs to check its owned by the player, as the rest of the code would check the object footprints for built on tiles